parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -71,6 +72,7 @@
 import org.apache.iceberg.deletes.EqualityDeleteWriter;
 import org.apache.iceberg.deletes.PositionDeleteWriter;
 import org.apache.iceberg.encryption.EncryptionKeyMetadata;
+import org.apache.iceberg.encryption.envelope.EnvelopeKeyMetadata;
```

Comment:
Can we move this class into `encryption`?

Why is it called "Envelope"? Isn't it just key metadata?

Reply:
> Can we move this class into encryption?

Sure.

> Why is it called "Envelope"? Isn't it just key metadata?

This form of key metadata keeps the wrapped keys, and the ID of the wrapping key. But we are unlikely to need any other form of key metadata in the future. We can call this class simply "KeyMetadata"